docs/cli/product-json-to-csv.md

```diff
@@ -0,0 +1,161 @@
+                                   categories column. [choices: "name", "key", "externalId",
+                                                          "namedPath"] [default: "name"]
+  --referenceCategoryOrderHintBy   Define which identifier should be used for the
+                                   categoryOrderHints column. [choices: "name", "key",
+                                   "externalId", "namedPath"] [default: "name"]
+  --fillAllRows                    Define if product attributes like name should be
+                                   added to each variant row.                  [boolean]
+  --language, -l                   Language used for localised attributes such as
+                                   category names.              [string] [default: "en"]
+  --delimiter, -d                  Used CSV delimiter.                    [default: ","]
+  --multiValueDelimiter, -m        Used CSV delimiter in multiValue fields. [default: ";"]
+  --encoding, -e                   Encoding used when saving data to output file
+                                                              [string] [default: "utf8"]
+  --logLevel                       Logging level: error, warn, info or debug    [string]
+                                                                       [default: "info"]
+  --prettyLogs                     Pretty print logs to the terminal           [boolean]
+  --logFile                        Path to file where logs should be saved      [string]
+                                                    [default: "product-json-to-csv.log"]
+```
+
+The products to be parsed from JSON to CSV can be passed to this module in one of two ways:
+- From a pipe
+- From a file
+
+#### Pass products through a pipe
+Piping products in JSON to be parsed. This ideally works with the commercetools product exporter. In this scenario, the products are parsed directly after export. More information on pipe streams can be found [here](http://www.gnu.org/software/bash/manual/bash.html#Pipelines)
+
+##### Example
+```
+$ @commercetools/product-exporter --projectKey <project_key> | @commercetools/product-json-to-csv \
+--projectKey <project_key> --template <path_to_template_file> --output <path_to_output_file>
+```
+
+#### Pass products from a file
+This module also accepts products to be read from a JSON file. This can be done by specifying the `--input` flag
+
+
+##### Example
+```
+$ @commercetools@commercetools/product-json-to-csv --projectKey <project_key> --input <path_to_JSON_file> --template <path_to_template_file> --output <path_to_output_file>
+```
+
+#### CSV Parser Template
+A parser template defines the content of the resulting parsed CSV file, by listing wanted product attribute names as header row. The header column values will be parsed and the resulting CSV file will contain corresponding attribute values of the exported products.
+```
+# only productType.name, the variant id and localized name (english) will be exported
```

Comment:
Is this correctly formatted?

Reply:
You mean it should be "product type name, ..."?

Reply:
It is. The `productType` field resolves the `productType.name`